assets/src/blocks/Spreadsheet/SpreadsheetFrontend.js

```diff
@@ -125,37 +125,36 @@ export class SpreadsheetFrontend extends Component {
       return placeholderData.rows;
     } else if ( this.state.loading === true || this.state.loading === null ) {
       return [];
-    } else {
-      return this.state.spreadSheetData.rows;
     }
+    return this.state.spreadSheetData.rows;
   }
 
   renderRows() {
     const rows = this.sortRows( this.filterMatchingRows( this.getRows() ), this.state.sortColumnIndex );
 
     return this.state.searchText.length >= 1 && rows.length === 0
       ? <tr>
-          <td colSpan="99999">
-            <div className='spreadsheet-empty-message'>
-              { __('No data matching your search.', 'planet4-blocks') }
-            </div>
-          </td>
-        </tr>
-      : rows.map((row, rowNumber) => (
-          <tr key={ rowNumber } data-order={ rowNumber }>
-            {
-              row.map((cell, cellIndex) => (
-                <td key={ cellIndex }>
-                  {
-                    this.state.searchText.length > 0
+        <td colSpan="99999">
+          <div className="spreadsheet-empty-message">
+            { __( 'No data matching your search.', 'planet4-blocks' ) }
+          </div>
+        </td>
+      </tr>
+      : rows.map( ( row, rowNumber ) => (
+        <tr key={ rowNumber } data-order={ rowNumber }>
+          {
+            row.map( ( cell, cellIndex ) => (
+              <td key={ cellIndex }>
+                {
+                  this.state.searchText.length > 0
                     ? HighlightMatches( cell, this.state.searchText )
```

Comment:
Since `/* eslint-disable new-cap */` is for this line, I think we should write it here inline